src/main/java/cz/smarteon/loxone/calendar/CalEntryPeriodYearly.java

```diff
@@ -76,4 +76,9 @@ public CalEntryPeriodYearly(
     public CalendarCommand<EmptyValue> createEntryCommand(){
         return new CalendarCommand<>(CREATE_ENTRY + name + "/" + operatingMode + "/" + CAL_MODE + "/" + startMonth + "/" + startDay + "/" + endMonth + "/" + endDay, EmptyValue.class);
     }
+
+    public CalendarCommand<EmptyValue> updateEntryCommand(LoxoneUuid uuid) {
+        String command = createEntryCommand().getCommand().replace(CalendarCommand.COMMAND_PREFIX + CREATE_ENTRY, UPDATE_ENTRY + uuid + "/");
```

Comment:
this is ugly hack, please find better how to share the relevant part of those commands